Tests/WithBlocksTests.Generated.cs

```diff
@@ -0,0 +1,762 @@
+﻿using System;
```

Comment:
wouldnt this be better as a paramterized test rather than a TT?

Reply:
Well, each test can have a different return type (ConsumerInfo, FieldInfo, MethodInfo) and each test calls a different method, so I'm not really seeing how this can be parameterized.

Reply:
yep good point